bin/live/pycbc_live_supervise_single_significance_fits

```diff
@@ -0,0 +1,516 @@
+    combined_plot_output = os.path.join(output_dir,
+                                        f"{{ifo}}-{out_fname}-{{type}}.png")
+    combined_plot_arguments = [
+        'pycbc_live_plot_combined_single_significance_fits',
+        '--combined-fits-file',
+        combined_options['output'],
+        '--output-plot-name-format',
+        combined_plot_output,
+        '--log-colormap'
+    ]
+
+    run_and_error(combined_plot_arguments)
+
+    combined_plot_outputs = [
+        combined_plot_output.format(ifo=ifo, type='fit_coeffs') for ifo in
+        combined_options['ifos'].split()
+    ]
+    combined_plot_outputs += [
+        combined_plot_output.format(ifo=ifo, type='counts') for ifo in
+        combined_options['ifos'].split()
+    ]
+
+    # Link the plots to the public-dir if wanted
+    if 'public-dir' in controls:
+        logging.info("Linking combined fits")
+        for cpo in combined_plot_outputs:
+            symlink(cpo, public_dir)
+
+    logging.info('Done')
+
+
+def wait_for_utc_time(target_str):
+    """Wait until the UTC time is as given by `target_str`, in HH:MM:SS format.
+    """
+    target_hour, target_minute, target_second = map(int, target_str.split(':'))
+    now = datetime.utcnow()
+    # for today's target, take now and replace the time
+    target_today = now + relativedelta(
+        hour=target_hour, minute=target_minute, second=target_second
+    )
+    # for tomorrow's target, take now, add one day, and replace the time
+    target_tomorrow = now + relativedelta(
+        days=1, hour=target_hour, minute=target_minute, second=target_second
+    )
+    next_target = target_today if now <= target_today else target_tomorrow
+    sleep_seconds = (next_target - now).total_seconds()
+    logging.info('Waiting %.0f s', sleep_seconds)
+    time.sleep(sleep_seconds)
+
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument('--config-file', required=True)
+parser.add_argument(
+    '--date',
+    help='Date to analyse, if not given, will analyse yesterday (UTC). '
+         'Format YYYY_MM_DD. Do not use if using --run-daily-at.'
+)
+parser.add_argument(
+    '--combine-only',
+    action='store_true',
+    help="Only do the combination of singles fit files."
+)
+parser.add_argument(
+    '--daily-only',
+    action='store_true',
+    help="Only do the daily singles fitting."
+)
+parser.add_argument(
+    '--run-daily-at',
+    metavar='HH:MM:SS',
+    help='Stay running and repeat the fitting daily at the given UTC hour.'
+)
+args = parser.parse_args()
+
+pycbc.init_logging(True)
+
+if args.run_daily_at is not None and args.date is not None:
+    parser.error('Cannot take --run-daily-at and --date at the same time')
+
+if args.run_daily_at is not None:
+    # keep running and repeat the fitting every day at the given hour
+    if not re.match('[0-9][0-9]:[0-9][0-9]:[0-9][0-9]', args.run_daily_at):
+        parser.error('--run-daily-at takes a UTC time in the format HH:MM:SS')
```

Comment:
After the discussion we had a few days ago on Slack, can you add this just before the infinite loop:
```python
    logging.info('Starting in daily run mode')
```